api/api.go

```diff
@@ -17,39 +18,62 @@ import (
 	"tier.run/values"
 )
 
+func init() {
+	// check that all error codes are unique
+	seen := map[string]bool{}
+	for _, e := range errorLookup {
+		if seen[e.Code] {
+			panic("duplicate error message")
+		}
+		if seen[e.Code] {
+			panic(fmt.Sprintf("duplicate error code %q", e.Code))
+		}
+		seen[e.Code] = true
+	}
+}
+
 // HTTP Errors
-var errorLookup = map[error]error{
-	control.ErrOrgNotFound: &trweb.HTTPError{
+//
+// TERR1000: invalid requests
+// TERR1010: org or features problem
+// TERR2000: internal errors
+var errorLookup = map[error]*trweb.HTTPError{
+	control.ErrOrgNotFound: {
 		Status:  400,
```

Comment:
400 is fine, but sort of feels like these "not found" errors should be 404s?

LGTM, just a bikeshed.

Reply:
If I POST `/v1/schedule` it's a 400, and I think it should stay that, but we can change it to a 404 for `/v1/whoami` if it is not that way already.